src/inspector/view/qml/MuseScore/Inspector/score/ScoreAccessibilityInspectorView.qml

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Inspector 1.0
+import Muse.UiComponents 1.0
+import Muse.Ui 1.0
+
+import "../common"
+
+InspectorSectionView {
+    id: root
+
+    implicitHeight: contentColumn.implicitHeight
+
+    Column {
+        id: contentColumn
+        width: parent.width
+        spacing: 8
+
+        Text {
+            text: qsTr("Score style preset")
+            color: ui.theme.fontPrimaryColor
+        }
+
+        StyledDropdown {
+            id: scoreStylePreset
+            width: parent.width
+            currentIndex: root.model ? root.model.scoreStylePresetIndex : 0
+            model: root.model ? root.model.possibleScoreStylePresets : []
+
+            Component.onCompleted: {
+                root.model.readStyleFileAccessibilityStyleEdited()
+            }
+
+            onActivated: function(index, value) {
+                if (!value.edited) {
+                    root.model.loadStyle(value.preset)
+                }
+            }
```

Comment:
I think it might be preferable to call `loadStyle` from C++ indeed. If you only change the index, but don't call `loadStyle`, things are in an inconsistent state, so you'll never want to update only the index. Therefore, always calling `loadStyle` from C++ seems good to me.

Reply:
Yeah makes sense. Moved the logic to the `setScoreStylePresetIndex(index)`